app/models/concerns/actions/targets_many.rb

```diff
@@ -33,7 +33,8 @@ def page_size
   end
 
   def remaining_targets
-    targeted.where("id > ?", last_completed_id).where.not(id: failed_ids).reorder(:id)
+    next_possible_target_id = last_completed_id + 1
+    targeted.where(id: next_possible_target_id..).where.not(id: failed_ids).reorder(:id)
```

Comment:
This will infer the table name, but produces a ">=" comparison. We add one to the last completed id to ensure it's excluded in the new range.